python/vineyard/data/dataset.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python
```

Comment:
Put the file to `python/vineyard/contrib` as `python/vineyard/contrib/pytorch.py`, as every ML framework has its own `DataSet`.

Reply:
Nope. I suppose this should be the "common" dataset type we would like to support in vineyard.

Reply:
But it inherits from pytorch.

Pytorch itself shouldn't be dependent of vineyard.